File: Practices/Pr12/prob12.java
import java.util.Scanner;

public class prob12 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        String name;

        while (!(name = in.nextLine()).equals("End")) {

            boolean lead = false;
            boolean flex = false;
            boolean involve = false;
            boolean plan = false;

            String line;
            while ( !(line = in.nextLine().toLowerCase()).equals("")) {
                if (line.contains("lead"))
                    lead = true;
                if (line.contains("flexib"))
                    flex = true;
                if (line.contains("involve"))
                    involve = true;
                if (line.contains("plan"))
                    plan = true;
            }

            if (lead && flex && involve && plan) {
                System.out.println(name + ": Meets standards");
            } else 
                System.out.println(name + ": Does not meet standards");
        }
    }
}

File: Practices/Pr12/prob12.py
text = []
while True:
    data = input()
    if data == "End":
        break
    else:
        place = data
    

    while True:
        data = input()
        if data == "":
            break
        else:
            text.extend(data.split())
    
    bunch = "".join(text).lower()

    if  "lead" in bunch and "flexib" in bunch and "involve" in bunch and "plan" in bunch:
        print(place + ": " + "Meets standards")
        # print(f"{place}: Meets standards")
    else:
        print(place + ": " + "Does not meet standards")
        # print(f"{place}: Does not meet standards")
    print()
    
    text = []